cantabular/client.go

```diff
@@ -178,7 +178,7 @@ func (c *Client) errorResponse(url string, res *http.Response) error {
 	}
 
 	if len(b) == 0 {
-		b = []byte("[response body empty]")
+		b = []byte(`{"message": "(error response body empty)"}`)
```

Comment:
To make this message clear that it's referring to an error response from cantabular it would be good to say something like this:
```suggestion
		b = []byte(`{"message": "(error response body empty)"}`)
```
```suggestion
		b = []byte(`{"message": "(cantabular error has empty response body)"}`)
```

Reply:
The way we wrap errors at each level this won't be necessary. It will always look like `failed to do a: failed to post query: {error response body empty}. We also don't want to mention cantabular in any responses we return to the user

Reply:
That makes sense, thanks.